crates/languages/src/diff/highlights.scm

```diff
@@ -18,8 +18,6 @@
   "diff" @function
   (argument) @variable.parameter)
 
-(filename) @string.special.path
```

Comment:
(filename) was conflicting with `(old_file)` and `(new_file)`.

If we choose to keep (filename), then we should remove others. Github does highlight the filename accordingly.